### DISCUSSION TITLE
Compiling Error - "pause_menu_response" missing?

### DISCUSSION THREAD

**Discussion by Celleforst** [Jyers/Marlin#762] [Q&A] [answered]
Hey,
I am trying to compile the printer software with Visual Code but got this error message:

Marlin\src\lcd\extui\creality_dwin\creality_dwin.cpp: In member function 'void CrealityDWINClass::Popup_Control()':
Compiling .pio\build\STM32F103RET6_creality\src\src\lcd\marlinui.cpp.o
Marlin\src\lcd\extui\creality_dwin\creality_dwin.cpp:4873:11: error: 'pause_menu_response' was not declared in this scope
           pause_menu_response = PAUSE_RESPONSE_EXTRUDE_MORE;
           ^~~~~~~~~~~~~~~~~~~
Marlin\src\lcd\extui\creality_dwin\creality_dwin.cpp:4873:11: note: suggested alternative: 'base_min_pos'
           pause_menu_response = PAUSE_RESPONSE_EXTRUDE_MORE;
           ^~~~~~~~~~~~~~~~~~~
           base_min_pos
Marlin\src\lcd\extui\creality_dwin\creality_dwin.cpp:4873:33: error: 'PAUSE_RESPONSE_EXTRUDE_MORE' was not declared in this scope
           pause_menu_response = PAUSE_RESPONSE_EXTRUDE_MORE;
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~
Marlin\src\lcd\extui\creality_dwin\creality_dwin.cpp:4876:11: error: 'pause_menu_response' was not declared in this scope
           pause_menu_response = PAUSE_RESPONSE_RESUME_PRINT;
           ^~~~~~~~~~~~~~~~~~~
Marlin\src\lcd\extui\creality_dwin\creality_dwin.cpp:4876:11: note: suggested alternative: 'base_min_pos'
           pause_menu_response = PAUSE_RESPONSE_RESUME_PRINT;
           ^~~~~~~~~~~~~~~~~~~
           base_min_pos

So basically this pause_menu_pause variable seems to be missing...
How can I fix this?

**Comment by Cal-LF** [comment, marked as answer]
Are you using one of the templates provided in the Configuration Files\E3V2 Templates? I compiled the 3X3 Man Mesh with success. yesterday.

**Reply by Celleforst** [reply]
I didn't realise that there were templates provided. Now I also managed to compile sucessfully with one of the templates.
Thanks for your help!
